index.html

```diff
@@ -4,9 +4,47 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Portfolio skeleton</title>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" 
+  integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" 
+  crossorigin="anonymous">  
+  <link rel="stylesheet" href="style.css">
+  <title>Portfolio_Website</title>
 </head>
 <body>
-    <h1>Designing Portfolio Skeleton</h1>
+  <!--Header Section -->
+
+  <main class="head-section"> 
+      <nav class="top d-flex">
+        <h1>My Logo</h1>
+        <i class="fas fa-bars"></i>
+      </nav>     
+
+     <!--Main Section -->
+
+     <section  class="main-section">
+      <h2 class="intro">I'm Ravi Teja<span>glad to see</span><span>
+      you!!</span></h2>
+       <article  class="description">
+          <p>I’m a software developer! I can help you build a product , 
+             feature or website Look through some of my work and experience! 
+             If you like but what you see and have a project you need coded, 
+             don’t hestiate to contact me.
+          </p>
+      </article>
+      
+      <section class="footer-section">
+        <h5>LET'S CONNECT</h5>      
+          <ul class="icons">
+            <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+            <li><a href="#"><i class="fab fa-linkedin-in"></i></a></li>
+            <li><a href="#"><img src="images/Vector.svg" alt="Vector" ></a></li>         
+            <li><a href="#"><i class="fab fa-github"></i></a></li>            
+            <li><a href="#"><img src="images/Finger.svg" alt="Finger"></a></li>
+          </ul>     
+      </section>  
+   </section>  
+   
+   </main>   
```

Comment:
- [ ] GOOD JOB :clap: :clap: I can see you have improved your styling you are almost there. you just have only to set the `font-family` as well as the `font-size`. Kindly follow the template style

| your design | the template|
| --------------- | ---------------- |
![Screenshot from 2021-11-26 04-38-02](https://user-images.githubusercontent.com/65068771/143518413-f13487e6-734c-4cf3-add6-5ba93c13b064.png) | ![143390039-f6c2a839-f2f9-4217-acae-05cf29dcefb2](https://user-images.githubusercontent.com/65068771/143435936-2c7eb81c-13b2-43c2-b449-77ae83affe45.png)